aws/environments/stable3.yml

```diff
@@ -22,3 +22,5 @@ use_rds: "true"
 
 content_static_resource_url: https://static-stable.dev.lcip.org
 auth_recovery_code_count: 3
+
+fxadev_git_version: fxa-mono-docker-image
```

Comment:
Note to self, delete this line before merging

Reply:
🤔

Reply:
I deleted it after merging >.>

https://github.com/mozilla/fxa-dev/commit/8fc5654b1803ef205c502bb77182b0dd9a50341c